src/module_2/explicacion_data_analysis.md

```diff
@@ -0,0 +1,425 @@
+    mean   3.690916e+13         62.532837              0.007128     6.553785   
+    std    2.695602e+12        151.179244              0.017232     6.490753   
+    min    3.361529e+13          1.000000              0.000114     0.570000   
+    25%    3.427657e+13          9.000000              0.001026     2.890000   
+    50%    3.927260e+13         27.000000              0.003078     4.490000   
+    75%    3.946662e+13         71.000000              0.008093     7.990000   
+    max    4.015515e+13       4487.000000              0.511456    59.990000   
+    
+           compare_at_price  
+    count       1477.000000  
+    mean           7.316168  
+    std            7.071117  
+    min            0.750000  
+    25%            3.000000  
+    50%            4.990000  
+    75%            8.970000  
+    max           60.000000  
+
+
+
+    
+![png](explicacion_data_analysis_files/explicacion_data_analysis_13_1.png)
+    
+
+
+En esta corr matrix yo no veo mucho y seguramente sea debido a que habia muchos products ids que aparecian en orders pero no en el inventario por lo que ahi perdiamos mucha info a la hora de mergear ambis( opté por no usarlos)
+
+
+```python
+freq_encoding_vendor_product_type(df_orders_combined_inventory)
+make_corr_matrix_and_statictics(df_orders_combined_inventory)
+```
+
+             variant_id  number_of_orders  purchase_probability        price  \
+    count  1.477000e+03       1477.000000           1477.000000  1477.000000   
+    mean   3.690916e+13         62.532837              0.007128     6.553785   
+    std    2.695602e+12        151.179244              0.017232     6.490753   
+    min    3.361529e+13          1.000000              0.000114     0.570000   
+    25%    3.427657e+13          9.000000              0.001026     2.890000   
+    50%    3.927260e+13         27.000000              0.003078     4.490000   
+    75%    3.946662e+13         71.000000              0.008093     7.990000   
+    max    4.015515e+13       4487.000000              0.511456    59.990000   
+    
+           compare_at_price  vendor_freq  product_type_freq  
+    count       1477.000000  1477.000000        1477.000000  
+    mean           7.316168     0.006820           0.033772  
+    std            7.071117     0.007879           0.024450  
+    min            0.750000     0.000677           0.001354  
+    25%            3.000000     0.002031           0.015572  
+    50%            4.990000     0.004062           0.023697  
+    75%            8.970000     0.008125           0.046716  
+    max           60.000000     0.037238           0.085985  
+
+
+
+    
+![png](explicacion_data_analysis_files/explicacion_data_analysis_15_1.png)
+    
+
+
+Aqui intente hacer un freq encoding con el vendedor y tipo de cada producto y   vemos una relaccio entre el tipo de producto y el precio asi como el tipo de producto y el vendor , ambas cosas tienen sentido
```

Comment:
What were your expectations with these plot?
In addition an id makes no sense in acorrelation plot